packages/nouns-webapp/src/hooks/useBlockTimestamp.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import { useEthers } from '@usedapp/core';
+
+export function useBlockTimestamp(blockno: number | undefined): number | undefined {
```

Comment:
Very clean. 

`nit` can we add a doc string here mirroring the format here (https://sourcegraph.com/github.com/nounsDAO/nouns-monorepo/-/blob/packages/nouns-webapp/src/utils/colorResponsiveUIUtils.ts?L11) ?

Reply:
Absolutely, good idea.